test/dashdb.replaceById.test.js

```diff
@@ -0,0 +1,114 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: loopback-connector-dashdb
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+/* eslint-env node, mocha */
+process.env.NODE_ENV = 'test';
+
+require('./init.js');
+const assert = require('assert');
+
+let id, db, Book;
+const BOOK_TITLE = 'Rocky I';
+const REVISED_BOOK_TITLE = 'Rocky II';
+
+/*
+   This test suite is to test the changes surrounding the fix
+   in loopback-ibmdb related to issue :
+   https://github.com/strongloop/loopback-ibmdb/issues/79
+*/
+
+describe('replaceById with BOOK_ID', function() {
+  // 'table' option specified since we want a table 'BOOK'
+  // to be created for model 'Book'
+  const BOOK_TABLE_OPTIONS = {
+    dashdb: {
+      table: 'BOOK',
+    },
+  };
+    // 'columnName' options specified since  want columns 'BOOK_ID' and 'TITLE'
+    // to be created instead of 'bookId' and 'title'
+  const BOOK_PROPERTIES = {
+    bookId: {
+      type: Number,
+      id: true,
+      generated: true,
+      dashdb: {
+        columnName: 'BOOK_ID',
+      },
+    },
+    title: {
+      type: String,
+      dashdb: {
+        columnName: 'TITLE',
+      },
+    },
+  };
+  before(function() {
+    db = global.getDataSource();
+    Book = db.define('Book', BOOK_PROPERTIES, BOOK_TABLE_OPTIONS);
+  });
+  it('create and replace a book', function(done) {
+    db.automigrate('Book', function(err) {
+      if (err) return done(err);
+      Book.create({title: BOOK_TITLE}, function(err, newBook) {
+        if (err) return done(err);
+        assert(newBook);
+        assert.equal(newBook.title, BOOK_TITLE);
+        id = newBook.id;
+        Book.replaceById(id, {title: REVISED_BOOK_TITLE},
+          function(err, updatedBook) {
+            if (err) return done(err);
+            assert(updatedBook);
+            assert.equal(updatedBook.id, id);
+            assert.equal(updatedBook.title, REVISED_BOOK_TITLE);
+            done();
+          });
+      });
+    });
+  });
```

Comment:
We are trying to move away from callback-based tests and use async/await wherever possible, unfortunately the effort to migrate the exist code base is large and therefore we haven't made the change yet. Let's not build up this technical debt by creating more callback-based code.

Can you please rewrite this test to async/await style?

```js
it('create and replace a book', async () => {
  await db.automigrate('Book');
  const newBook = await Book.create({title: BOOK_TITLE});
  assert(newBook);
  // and so on
});
```

The same comment applies to other newly added tests too.